index.html

```diff
@@ -11,51 +11,183 @@
+        <img src="images/Img_Plaholder.svg" alt="" class="imgPlaceholder">
+        <h2 class="projectTitle">Multi-Post Stories Gain+Glory</h2>
+        <ul class="langContainer">
+          <li class="lang"><a class="lang" href="">Ruby on rails</a></li>
+          <li class="lang"><a class="lang" href="">css</a></li>
+          <li class="lang"><a class="lang" href="">JavScript</a></li>
+          <li class="lang"><a class="lang" href="">html</a></li>
+        </ul>
+        <button type="button" class="seeBtn">See Project</button>
+      </article>
+    </div>
+    <div class="gridCell work">
+      <article class="cardContainer">
+        <img src="images/Img_Plaholder.svg" alt="" class="imgPlaceholder">
+        <h2 class="projectTitle">Multi-Post Stories Gain+Glory</h2>
+        <ul class="langContainer">
+          <li class="lang"><a class="lang" href="">Ruby on rails</a></li>
+          <li class="lang"><a class="lang" href="">css</a></li>
+          <li class="lang"><a class="lang" href="">JavScript</a></li>
+          <li class="lang"><a class="lang" href="">html</a></li>
+        </ul>
+        <button type="button" class="seeBtn">See Project</button>
+      </article>
+    </div>
+  </section>
+  <article class="aboutMeBox">
+    <h1 class="h1aboutMe">About me
+    </h1>
+    <p class="p_aboutMe">I can help you build a product , feature or website Look through some of my work and
+      experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+      <button type="button" class="aboutMeBtn seeBtn">Get My Resume</button>
+  </article>
+  <div class="aboutMeCardCont">
+  <section class="gridContainerAM">
+    <div class="cardAM">
+      <img src="images/icon-languages.svg" alt="" class="imageAmCard">
+      <h1 class="cardAMTitle">Languages</h1>
+      <ul class="langList">
+        <li class="lang langName"><a class="lang langName" href="">JavaScript</a></li>
+        <li class="lang langName"><a class="lang langName" href="">Ruby</a></li>
+        <li class="lang langName"><a class="lang langName" href="">Html </a></li>
+        <li class="lang langName"><a class="lang langName" href="">CSS</a></li>
+      </ul>
+    </div>
+    <div class="cardAM">
+      <img src="images/icon-frameworks.svg" alt="" class="imageAmCard">
+      <h1 class="cardAMTitle">Frameworks</h1>
+      <ul class="langList">
+        <li class="lang langName"><a class="lang langName" href="">Bootstrap</a></li>
+        <li class="lang langName"><a class="lang langName" href="">Ruby on rails</a></li>
+        <li class="lang langName"><a class="lang langName" href="">RSpec</a></li>
+        <li class="lang langName"><a class="lang langName" href="">Capybara</a></li>
+        <li class="lang langName"><a class="lang langName" href="">Selenium</a></li>
+      </ul>
+    </div>
+    <div class="cardAM">
+      <img src="images/icon-skills.svg" alt="" class="imageAmCard">
+      <h1 class="cardAMTitle">Skills</h1>
```

Comment:
same here please change to a `hx` other than `h1`

Reply:
- [ ] This issue is well resolved :heavy_check_mark: